### ISSUE TITLE
Add sdk/metric/view structure to new_sdk/main

### ISSUE THREAD

**Issue by MrAlias** [open-telemetry/opentelemetry-go#2817]
~Blocked by #2799~

- [ ] Build the unimplemented structure of the `sdk/metric/view` package.
- [ ] Ensure all implementation `TODO`s have an issue tracking them.

**Comment by MrAlias** [comment]
The current `views` package in the new_sdk/example branch has the following types:

```go
type (
	View struct {
		cfg Config
	}

	Config struct {
		...
	}
)
```

I would propose we rename the package to `view` and just export a `Config` type. That way `func WithViews(views ...view.Config) Option` in the `sdk/metric` reads without a stutter. Additionally, if we then have `New(...Option) Config` in the `view` package the call would be `WithViews(view.New(...), view.New(...)` and this would match our existing configuration patterns.

**Comment by Aneurysm9** [comment]
We use functional options for this sort of configuration pretty much everywhere else.  It would feel incongruous to not do so here.

**Comment by MrAlias** [comment]
Closed by #2838
